src/components/common/Dialogs/RaiseObjectionDialog/ObjectionAnnotation.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { DialogSection } from '~shared/Dialog';
+import { Annotations } from '~shared/Fields';
+
+const displayName = 'common.Dialogs.RaiseObjectionDialog.ObjectionAnnotation';
+
+const MSG = defineMessages({
+  annotation: {
+    id: `${displayName}.annotation`,
+    defaultMessage: `Explain why you’re making this objection (optional)`,
+  },
+});
+
+interface ObjectionAnnotationProps {
+  disabled: boolean;
+}
+
+const ObjectionAnnotation = ({ disabled }: ObjectionAnnotationProps) => (
+  <DialogSection appearance={{ border: 'top' }}>
+    <Annotations
+      label={MSG.annotation}
+      name="annotation"
+      maxLength={4000}
+      disabled={disabled}
+    />
+  </DialogSection>
+);
```

Comment:
I forgot about this. 

Originally we've used IPFS for this, but hopefully is going to be saved in the DB.

Also, as improvement, I think the details box length (4000 in this case) should be a central constant value that is set in all places where we have annotations (although this can be added as a separate, improvement issue)

Reply:
https://github.com/JoinColony/colonyCDapp/issues/355